tests/scripts/webhook-create-signed-cert.sh

```diff
@@ -26,6 +26,16 @@ EOF
     exit 1
 }
 
+checkKubectlVersion() {
```

Comment:
Were you seeing integration tests fail locally, or what was the need for this?

Reply:
Yes I the tests were failing locally and I didn't really understand why from the logs; the cause was found to be from a version mismatch, so I added this check.

Reply:
Please make this a specific commit since it's unrelated to the main implemenation.